backend/scripts/seed_db.sh

```
@@ -41,6 +45,29 @@ parse_arguments() {
   fi
 }
 
+create_mock_secrets() {
+  local app_dir=$1
+  local secrets_file="${app_dir}.app.secrets.json"
+  local app_json="${app_dir}app.json"
+  local app_name=$(basename "$app_dir")
+
+  if [ "$USE_MOCK" = true ] && grep -q "oauth2" "$app_json"; then
+    # Extract client_id and client_secret variable names from app.json
+    local client_id_var=$(grep -o '"client_id": *"[^{]*{{ *[^}]*}}' "$app_json" | sed 's/.*{{ *\([^}]*\) *}}.*/\1/' | tr -d ' ')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)